pkg/crc/network/utils.go

```diff
@@ -125,6 +126,11 @@ func CheckCRCLocalDNSReachableFromHost(bundle *bundle.CrcBundleInfo, expectedIP
 		appsHostname := bundle.GetAppHostname("foo")
 		ip, err = net.LookupIP(appsHostname)
 		if err != nil {
+			// Right now goodhosts fallback is not implemented in windows so
+			// this checks should still return the error.
+			if crcos.CurrentOS() == crcos.WINDOWS {
+				return err
+			}
```

Comment:
@praveenkumar Just for clarification, earlier if this failed for windows it didn't error out, but now we will, (even though it doesn't have the `goodhosts` workaround) this will prevent me from a successful `crc start` right? earlier i could start crc and manually add entries to resolve the app domains..

Reply:
No, we made this change with #1136 and in windows since we don't have goodhosts implementation yet to add the entry to hosts file we will fall back to our earlier default way where we error out as soon as the dns resolution not works.

Reply:
Okay, thanks for the clarification!